### DISCUSSION TITLE
Course says private keys should start with "0x"

### DISCUSSION THREAD

**Discussion by Noveleader** [smartcontractkit/full-blockchain-solidity-course-py#1314] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/91677627/159204546-4f220b1f-1838-4c13-baee-39a7e81b8a1a.png)
But while working with ganache it shows this as a private key and in the lesson it somehow starts with ```0x```

**Comment by Noveleader** [comment, marked as answer]
We need to add "0x" by ourselves.
So for example if your private key is : ```7fbfa9b1b42185223cf9ac09266270659ff1a625a66bcf260999a09ae161b89a```
So you should write : ```0x7fbfa9b1b42185223cf9ac09266270659ff1a625a66bcf260999a09ae161b89a```

**Reply by cromewar** [reply]
Nice catch @Noveleader

**Reply by Noveleader** [reply]
> Nice catch @Noveleader

Thanks!
